Makefile

```diff
@@ -17,7 +17,7 @@ build:
 	go build $(FLAGS) -o bin/$(BIN_NAME)_$(VERSION) main.go
 
 test:
-	cd test && go test -v -timeout 120m -count=1
+	go test -v -timeout 120m -count=1 ./...
```

Comment:
**[Question]** Not to sure what the intend of the the directory changes here are, seems like its going up two levels from the current context. Could I grab some clarification on this?

Reply:
This will run every test in all directories and sub directories. Previously we only had tests in `test` folder. Now we have two levels of tests - acceptance tests in `acc_tests` folder and unit tests in `instaclustr` folder (this is because unit tests are used to test the internal methods used for CRUD but they are unexported so the tests need to be in the same package). Essentially, `./...` will run every test in the repository.

Reply:
Ah ok, gotcha. Sounds good.